./mvnw package -Dpackaging=docker fails with Plugin not found: com.google.cloud.tools:jib-maven-plugin

```
./mvnw package -Dpackaging=docker
WARNING: A restricted method in java.lang.System has been called
WARNING: java.lang.System::load has been called by org.fusesource.jansi.internal.JansiLoader in an unnamed module (file:/Users/sdelamo/.m2/wrapper/dists/apache-maven-3.9.15-bin/4faaaa08/apache-maven-3.9.15/lib/jansi-2.4.3.jar)
WARNING: Use --enable-native-access=ALL-UNNAMED to avoid a warning for callers in this module
WARNING: Restricted methods will be blocked in a future release unless native access is enabled

[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for example.micronaut:foo:docker:0.1
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: com.vladsch.flexmark:flexmark-html2md-converter:jar -> version ${html2md.converter.version} vs ${flexmark.version} @ io.micronaut.platform:micronaut-platform:5.0.0-SNAPSHOT, /Users/sdelamo/.m2/repository/io/micronaut/platform/micronaut-platform/5.0.0-SNAPSHOT/micronaut-platform-5.0.0-SNAPSHOT.pom, line 3099, column 19
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO] 
[INFO] -----------------------< example.micronaut:foo >------------------------
[INFO] Building foo 0.1
[INFO]   from pom.xml
[INFO] -------------------------------[ docker ]-------------------------------
[INFO] 
[INFO] --- enforcer:3.6.2:enforce (micronaut-enforce) @ foo ---
[INFO] Rule 0: org.apache.maven.enforcer.rules.version.RequireJavaVersion passed
[INFO] Rule 1: io.micronaut.maven.enforcer.CheckSnakeYaml passed
[INFO] 
[INFO] --- mn:5.0.0-M2:generate-openapi-generic (default-generate-openapi-generic) @ foo ---
[INFO] 
[INFO] --- mn:5.0.0-M2:generate-openapi-client (default-generate-openapi-client) @ foo ---
[INFO] 
[INFO] --- mn:5.0.0-M2:generate-openapi-server (default-generate-openapi-server) @ foo ---
[INFO] 
[INFO] --- mn:5.0.0-M2:generate-jsonschema (default-generate-jsonschema) @ foo ---
[INFO] 
[INFO] --- resources:3.5.0:resources (default-resources) @ foo ---
[INFO] Copying 2 resources from src/main/resources to target/classes
[INFO] 
[INFO] --- compiler:3.15.0:compile (default-compile) @ foo ---
[INFO] Nothing to compile - all classes are up to date.
[INFO] 
[INFO] --- resources:3.5.0:testResources (default-testResources) @ foo ---
[INFO] skip non existing resourceDirectory /private/var/folders/65/67w3r16d2cn0jmpygf_mgch80000gn/T/maven13799409805608396967/src/test/resources
[INFO] 
[INFO] --- compiler:3.15.0:testCompile (default-testCompile) @ foo ---
[INFO] Nothing to compile - all classes are up to date.
[INFO] 
[INFO] --- mn:5.0.0-M2:start-testresources-service (default-start-testresources-service) @ foo ---
[INFO] 
[INFO] --- mn:5.0.0-M2:validate-test-configuration (default-validate-test-configuration) @ foo ---
[INFO] 
[INFO] --- surefire:3.5.5:test (default-test) @ foo ---
[INFO] Using auto detected provider org.apache.maven.surefire.junitplatform.JUnitPlatformProvider
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running example.micronaut.FooTest
09:42:52.041 [main] INFO  i.m.context.env.DefaultEnvironment - Established active environments: [test]
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.328 s -- in example.micronaut.FooTest
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] 
[INFO] --- mn:5.0.0-M2:stop-testresources-service (default-stop-testresources-service) @ foo ---
[INFO] 
[INFO] --- mn:5.0.0-M2:validate-configuration (default-validate-configuration) @ foo ---
[INFO] 
[INFO] --- mn:5.0.0-M2:aot-analysis (default-aot-analysis) @ foo ---
[INFO] 
[INFO] --- mn:5.0.0-M2:docker (default-docker) @ foo ---
[INFO] Using JDK version 25 from maven.compiler.release
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.006 s
[INFO] Finished at: 2026-04-29T09:42:52+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.micronaut.maven:micronaut-maven-plugin:5.0.0-M2:docker (default-docker) on project foo: Plugin not found: com.google.cloud.tools:jib-maven-plugin -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

Sample app: 

[Archive.zip](https://github.com/user-attachments/files/27193671/Archive.zip)

--- a/micronaut-maven-core/src/main/java/io/micronaut/maven/services/ExecutorService.java
+++ b/micronaut-maven-core/src/main/java/io/micronaut/maven/services/ExecutorService.java
@@ -141,16 +141,12 @@ public class ExecutorService {
         if (pluginKey.equals(plugin.getKey())) {
             return true;
         }
-        String groupId = plugin.getGroupId();
         String artifactId = plugin.getArtifactId();
-        String version = plugin.getVersion();
-        if (groupId == null || artifactId == null) {
+        if (artifactId == null) {
             return false;
         }
-        if (pluginKey.equals(groupId + ":" + artifactId)) {
-            return true;
-        }
-        return version != null && pluginKey.equals(groupId + ":" + artifactId + ":" + version);
+        String version = plugin.getVersion();
+        return version != null && pluginKey.equals(plugin.getKey() + ":" + version);
     }
 
     /**

--- a/micronaut-maven-core/src/main/java/io/micronaut/maven/services/ExecutorService.java
+++ b/micronaut-maven-core/src/main/java/io/micronaut/maven/services/ExecutorService.java
@@ -90,7 +90,7 @@ public class ExecutorService {
 
     public final void executeGoal(MavenProject project, String pluginKey, String goal, Xpp3Dom overriddenConfiguration) throws MojoExecutionException {
         MavenProject targetProject = project == null ? mavenProject : project;
-        Plugin plugin = targetProject.getPlugin(pluginKey);
+        Plugin plugin = resolveBuildPlugin(targetProject, pluginKey);
         if (plugin != null) {
             String goalName = goal;
             AtomicReference<String> executionId = new AtomicReference<>(goalName);
@@ -117,6 +117,40 @@ public class ExecutorService {
         } else {
             throw new MojoExecutionException("Plugin not found: " + pluginKey);
         }
+    }
+
+    static Plugin resolveBuildPlugin(MavenProject targetProject, String pluginKey) {
+        Plugin plugin = targetProject.getPlugin(pluginKey);
+        if (plugin != null) {
+            return plugin;
+        }
+        if (targetProject.getBuild() == null || targetProject.getBuild().getPluginManagement() == null) {
+            return null;
+        }
+        return targetProject.getBuild().getPluginManagement().getPlugins()
+            .stream()
+            .filter(managedPlugin -> matchesPluginKey(managedPlugin, pluginKey))
+            .findFirst()
+            .orElse(null);
+    }
+
+    private static boolean matchesPluginKey(Plugin plugin, String pluginKey) {
+        if (plugin == null) {
+            return false;
+        }
+        if (pluginKey.equals(plugin.getKey())) {
+            return true;
+        }
+        String groupId = plugin.getGroupId();
+        String artifactId = plugin.getArtifactId();
+        String version = plugin.getVersion();
+        if (groupId == null || artifactId == null) {
+            return false;
+        }
+        if (pluginKey.equals(groupId + ":" + artifactId)) {
+            return true;
+        }
+        return version != null && pluginKey.equals(groupId + ":" + artifactId + ":" + version);
     }
 
     /**

--- a/micronaut-maven-core/src/test/java/io/micronaut/maven/services/ExecutorServiceTest.java
+++ b/micronaut-maven-core/src/test/java/io/micronaut/maven/services/ExecutorServiceTest.java
@@ -3,6 +3,9 @@ package io.micronaut.maven.services;
 import org.apache.maven.artifact.repository.ArtifactRepository;
 import org.apache.maven.execution.MavenExecutionRequest;
 import org.apache.maven.execution.MavenSession;
+import org.apache.maven.model.Build;
+import org.apache.maven.model.Plugin;
+import org.apache.maven.model.PluginManagement;
 import org.apache.maven.plugin.BuildPluginManager;
 import org.apache.maven.project.MavenProject;
 import org.apache.maven.shared.invoker.InvocationRequest;
@@ -96,13 +99,54 @@ class ExecutorServiceTest {
         Path projectDirectory = tempDir.resolve("project");
         Path buildDirectory = projectDirectory.resolve("target");
         MavenProject project = new MavenProject();
-        org.apache.maven.model.Build build = new org.apache.maven.model.Build();
+        Build build = new Build();
         build.setDirectory(buildDirectory.toString());
         project.setBuild(build);
         File alternatePom = projectDirectory.resolve("custom.xml").toFile();
         project.setFile(alternatePom);
 
         assertSame(alternatePom, ExecutorService.resolveOriginalPom(project));
+    }
+
+    @Test
+    void resolveBuildPluginFallsBackToManagedPlugin() {
+        MavenProject project = new MavenProject();
+        Build build = new Build();
+        PluginManagement pluginManagement = new PluginManagement();
+        Plugin plugin = new Plugin();
+        plugin.setGroupId("com.google.cloud.tools");
+        plugin.setArtifactId("jib-maven-plugin");
+        plugin.setVersion("3.5.1");
+        pluginManagement.addPlugin(plugin);
+        build.setPluginManagement(pluginManagement);
+        project.setBuild(build);
+
+        Plugin result = ExecutorService.resolveBuildPlugin(project, "com.google.cloud.tools:jib-maven-plugin");
+
+        assertSame(plugin, result);
+    }
+
+    @Test
+    void resolveBuildPluginPrefersDirectBuildPluginOverManagedPlugin() {
+        MavenProject project = new MavenProject();
+        Build build = new Build();
+        Plugin directPlugin = new Plugin();
+        directPlugin.setGroupId("com.google.cloud.tools");
+        directPlugin.setArtifactId("jib-maven-plugin");
+        directPlugin.setVersion("3.5.1");
+        build.addPlugin(directPlugin);
+        PluginManagement pluginManagement = new PluginManagement();
+        Plugin managedPlugin = new Plugin();
+        managedPlugin.setGroupId("com.google.cloud.tools");
+        managedPlugin.setArtifactId("jib-maven-plugin");
+        managedPlugin.setVersion("3.5.0");
+        pluginManagement.addPlugin(managedPlugin);
+        build.setPluginManagement(pluginManagement);
+        project.setBuild(build);
+
+        Plugin result = ExecutorService.resolveBuildPlugin(project, "com.google.cloud.tools:jib-maven-plugin");
+
+        assertSame(directPlugin, result);
     }
 
     private static MavenSession newSession(Path tempDir, boolean quiet, File settingsFile) {

--- a/micronaut-maven-core/src/test/java/io/micronaut/maven/services/ExecutorServiceTest.java
+++ b/micronaut-maven-core/src/test/java/io/micronaut/maven/services/ExecutorServiceTest.java
@@ -113,10 +113,7 @@ class ExecutorServiceTest {
         MavenProject project = new MavenProject();
         Build build = new Build();
         PluginManagement pluginManagement = new PluginManagement();
-        Plugin plugin = new Plugin();
-        plugin.setGroupId("com.google.cloud.tools");
-        plugin.setArtifactId("jib-maven-plugin");
-        plugin.setVersion("3.5.1");
+        Plugin plugin = plugin("com.google.cloud.tools", "jib-maven-plugin", "3.5.1");
         pluginManagement.addPlugin(plugin);
         build.setPluginManagement(pluginManagement);
         project.setBuild(build);
@@ -130,16 +127,10 @@ class ExecutorServiceTest {
     void resolveBuildPluginPrefersDirectBuildPluginOverManagedPlugin() {
         MavenProject project = new MavenProject();
         Build build = new Build();
-        Plugin directPlugin = new Plugin();
-        directPlugin.setGroupId("com.google.cloud.tools");
-        directPlugin.setArtifactId("jib-maven-plugin");
-        directPlugin.setVersion("3.5.1");
+        Plugin directPlugin = plugin("com.google.cloud.tools", "jib-maven-plugin", "3.5.1");
         build.addPlugin(directPlugin);
         PluginManagement pluginManagement = new PluginManagement();
-        Plugin managedPlugin = new Plugin();
-        managedPlugin.setGroupId("com.google.cloud.tools");
-        managedPlugin.setArtifactId("jib-maven-plugin");
-        managedPlugin.setVersion("3.5.0");
+        Plugin managedPlugin = plugin("com.google.cloud.tools", "jib-maven-plugin", "3.5.0");
         pluginManagement.addPlugin(managedPlugin);
         build.setPluginManagement(pluginManagement);
         project.setBuild(build);
@@ -147,6 +138,56 @@ class ExecutorServiceTest {
         Plugin result = ExecutorService.resolveBuildPlugin(project, "com.google.cloud.tools:jib-maven-plugin");
 
         assertSame(directPlugin, result);
+    }
+
+    @Test
+    void resolveBuildPluginReturnsNullWithoutPluginManagement() {
+        MavenProject project = new MavenProject();
+
+        assertNull(ExecutorService.resolveBuildPlugin(project, "com.google.cloud.tools:jib-maven-plugin"));
+
+        project.setBuild(new Build());
+
+        assertNull(ExecutorService.resolveBuildPlugin(project, "com.google.cloud.tools:jib-maven-plugin"));
+    }
+
+    @Test
+    void resolveBuildPluginIgnoresNullIncompleteAndNonMatchingManagedPlugins() {
+        MavenProject project = new MavenProject();
+        Build build = new Build();
+        PluginManagement pluginManagement = new PluginManagement();
+        pluginManagement.getPlugins().add(null);
+        pluginManagement.addPlugin(plugin("com.google.cloud.tools", null, "3.5.1"));
+        pluginManagement.addPlugin(plugin("com.google.cloud.tools", "other-maven-plugin", null));
+        build.setPluginManagement(pluginManagement);
+        project.setBuild(build);
+
+        Plugin result = ExecutorService.resolveBuildPlugin(project, "com.google.cloud.tools:jib-maven-plugin");
+
+        assertNull(result);
+    }
+
+    @Test
+    void resolveBuildPluginMatchesManagedPluginWithVersionedKey() {
+        MavenProject project = new MavenProject();
+        Build build = new Build();
+        PluginManagement pluginManagement = new PluginManagement();
+        Plugin plugin = plugin("com.google.cloud.tools", "jib-maven-plugin", "3.5.1");
+        pluginManagement.addPlugin(plugin);
+        build.setPluginManagement(pluginManagement);
+        project.setBuild(build);
+
+        Plugin result = ExecutorService.resolveBuildPlugin(project, "com.google.cloud.tools:jib-maven-plugin:3.5.1");
+
+        assertSame(plugin, result);
+    }
+
+    private static Plugin plugin(String groupId, String artifactId, String version) {
+        Plugin plugin = new Plugin();
+        plugin.setGroupId(groupId);
+        plugin.setArtifactId(artifactId);
+        plugin.setVersion(version);
+        return plugin;
     }
 
     private static MavenSession newSession(Path tempDir, boolean quiet, File settingsFile) {

--- a/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/invoker.properties
+++ b/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/invoker.properties
@@ -1,0 +1,1 @@
+invoker.goals = -ntp package -Dpackaging=docker -Djib.buildGoal=buildTar

--- a/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/pom.xml
+++ b/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/pom.xml
@@ -1,0 +1,133 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.micronaut.build.examples</groupId>
+    <artifactId>package-docker-managed-jib</artifactId>
+    <version>0.1</version>
+    <packaging>${packaging}</packaging>
+
+    <parent>
+        <groupId>io.micronaut.platform</groupId>
+        <artifactId>micronaut-parent</artifactId>
+        <version>@it.micronaut.version@</version>
+    </parent>
+
+    <properties>
+        <micronaut.version>@it.micronaut.version@</micronaut.version>
+        <micronaut-maven-plugin.version>@project.version@</micronaut-maven-plugin.version>
+        <exec.mainClass>io.micronaut.build.examples.Application</exec.mainClass>
+        <packaging>jar</packaging>
+        <micronaut.runtime>netty</micronaut.runtime>
+        <maven.native.plugin.version>@native-maven-plugin.version@</maven.native.plugin.version>
+        <jib-maven-plugin.version>@jib-maven-plugin.version@</jib-maven-plugin.version>
+        <maven.compiler.release>${java.specification.version}</maven.compiler.release>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.micronaut</groupId>
+            <artifactId>micronaut-http-server-netty</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.micronaut.serde</groupId>
+            <artifactId>micronaut-serde-jackson</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.micronaut</groupId>
+            <artifactId>micronaut-http-client</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.micronaut.test</groupId>
+            <artifactId>micronaut-test-junit5</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>com.google.cloud.tools</groupId>
+                    <artifactId>jib-maven-plugin</artifactId>
+                    <version>${jib-maven-plugin.version}</version>
+                    <configuration>
+                        <from>
+                            <platforms>
+                                <platform>
+                                    <architecture>amd64</architecture>
+                                    <os>linux</os>
+                                </platform>
+                            </platforms>
+                        </from>
+                        <to>
+                            <image>docker.io/alvarosanchez/${project.artifactId}:${project.version}</image>
+                        </to>
+                        <container>
+                            <mainClass>${exec.mainClass}</mainClass>
+                        </container>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>io.micronaut.maven</groupId>
+                <artifactId>micronaut-maven-plugin</artifactId>
+                <version>${micronaut-maven-plugin.version}</version>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <!-- Uncomment to enable incremental compilation -->
+                    <!-- <useIncrementalCompilation>false</useIncrementalCompilation> -->
+                    <annotationProcessorPaths combine.children="append">
+                        <path>
+                            <groupId>io.micronaut</groupId>
+                            <artifactId>micronaut-http-validation</artifactId>
+                            <version>${micronaut.core.version}</version>
+                        </path>
+                        <path>
+                            <groupId>io.micronaut.serde</groupId>
+                            <artifactId>micronaut-serde-processor</artifactId>
+                            <version>${micronaut.serialization.version}</version>
+                        </path>
+                    </annotationProcessorPaths>
+                    <compilerArgs>
+                        <arg>-Amicronaut.processing.group=io.micronaut.build.examples</arg>
+                        <arg>-Amicronaut.processing.module=${project.artifactId}</arg>
+                    </compilerArgs>
+                </configuration>
+            </plugin>
+
+        </plugins>
+    </build>
+
+</project>

--- a/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/src/main/java/io/micronaut/build/examples/Application.java
+++ b/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/src/main/java/io/micronaut/build/examples/Application.java
@@ -1,0 +1,9 @@
+package io.micronaut.build.examples;
+
+import io.micronaut.runtime.Micronaut;
+
+public class Application {
+    public static void main(String[] args) {
+        Micronaut.run(Application.class);
+    }
+}

--- a/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/src/main/resources/application.yml
+++ b/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/src/main/resources/application.yml
@@ -1,0 +1,3 @@
+micronaut:
+  application:
+    name: package-docker-managed-jib

--- a/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/src/main/resources/logback.xml
+++ b/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/src/main/resources/logback.xml
@@ -1,0 +1,15 @@
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <withJansi>false</withJansi>
+        <!-- encoders are assigned the type
+             ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+        <encoder>
+            <pattern>%cyan(%d{HH:mm:ss.SSS}) %gray([%thread]) %highlight(%-5level) %magenta(%logger{36}) - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="info">
+        <appender-ref ref="STDOUT" />
+    </root>
+</configuration>

--- a/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/src/test/java/io/micronaut/build/examples/ApplicationTest.java
+++ b/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/src/test/java/io/micronaut/build/examples/ApplicationTest.java
@@ -1,0 +1,21 @@
+package io.micronaut.build.examples;
+
+import io.micronaut.runtime.EmbeddedApplication;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import jakarta.inject.Inject;
+
+@MicronautTest
+public class ApplicationTest {
+
+    @Inject
+    EmbeddedApplication application;
+
+    @Test
+    void testItWorks() {
+        Assertions.assertTrue(application.isRunning());
+    }
+
+}

--- a/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/verify.groovy
+++ b/micronaut-maven-integration-tests/src/it/package-docker-managed-jib/verify.groovy
@@ -1,0 +1,5 @@
+File log = new File(basedir, 'build.log')
+assert log.exists()
+assert log.text.contains("Using base image: eclipse-temurin:${System.getProperty("java.specification.version")}-jre")
+assert new File(basedir, 'target/jib-image.tar').exists()
+assert !log.text.contains("Plugin not found: com.google.cloud.tools:jib-maven-plugin")

